src/views/templates/list/hooks/useVirtualMachineTemplatesCPUMemory.tsx

```diff
@@ -1,23 +1,17 @@
-import * as React from 'react';
+import React, { ReactNode } from 'react';
 
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { V1Template } from '@kubevirt-utils/models';
-import {
-  getFlavorData,
-  getTemplateVirtualMachineCPU,
-  vCPUCount,
-} from '@kubevirt-utils/resources/template';
+import { getTemplateVirtualMachineObject, vCPUCount } from '@kubevirt-utils/resources/template';
+import { getMemoryCPU } from '@kubevirt-utils/resources/vm';
 import { readableSizeUnit } from '@kubevirt-utils/utils/units';
 import { isTemplateParameter } from '@kubevirt-utils/utils/utils';
 import { Popover, PopoverPosition } from '@patternfly/react-core';
 import { HelpIcon } from '@patternfly/react-icons';
 
-export const useVirtualMachineTemplatesCPUMemory = (
-  template: V1Template,
-): React.ReactNode | string => {
+export const useVirtualMachineTemplatesCPUMemory = (template: V1Template): ReactNode | string => {
```

Comment:
Hi :) 
A little suggestion: Making this a component instead of a hook would be better suited here it returns a component, Please consider to change it.

Reply:
I'd better delete that popover related code as I think it is deprecated. I was never able to find any scenario to see that code in the UI, I also have no idea why it is there. But I've found this PR: https://github.com/kubevirt-ui/kubevirt-plugin/pull/331
It looks like that additional code with adding popover to this hook was used to display in the VM Templates list. Now when I look at templates list, it looks like this:
![vmt](https://github.com/kubevirt-ui/kubevirt-plugin/assets/13417815/dfd852cd-e9f8-4a63-91a3-20c00a872006)
=> no any popover in the list, regarding CPU, Memory, no matter the Red Hat template or user created one, it looks the same. IMO someone just forgot to delete that code when the UI was changed. WDYT about it?

Reply:
If you're sure that this code is not needed than +1 on removing it

Reply:
Checked with Yifat also and it seems we really can remove that old code :)